ui/testem.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   tap_failed_tests_only: true,
   disable_watching: true,
   launch_in_ci: ['Chrome'],
+  launch_in_dev: ['Chrome'],
```

Comment:
Whoops, I didn't mean to commit this, though it is helpful if you want to run the full suite in the browser.